src/test/unit/AllocationManagerUnit.t.sol

```diff
@@ -1875,6 +1873,118 @@ contract AllocationManagerUnitTests_ClearModificationQueue is AllocationManagerU
+        uint32 deallocationEffectTimestamp = uint32(block.timestamp + DEALLOCATION_DELAY);
+        IAllocationManagerTypes.MagnitudeAllocation[] memory firstDeallocation =
+            _generateMagnitudeAllocationCalldataForOpSet(defaultAVS, 1, 25e16, 1e18);
+        cheats.prank(defaultOperator);
+        allocationManager.modifyAllocations(firstDeallocation);
+        MagnitudeInfo[] memory mInfos = allocationManager.getAllocationInfo(defaultOperator, strategyMock, firstDeallocation[0].operatorSets);
+        assertEq(deallocationEffectTimestamp, mInfos[0].effectTimestamp, "effect timestamp not correct");
+        
+        // Allocate 33e16 mag to opset2
+        uint32 allocationEffectTimestamp = uint32(block.timestamp + allocationDelay);
+        IAllocationManagerTypes.MagnitudeAllocation[] memory secondAllocation =
+            _generateMagnitudeAllocationCalldataForOpSet(defaultAVS, 2, 33e16, 1e18);
+        cheats.prank(defaultOperator);
+        allocationManager.modifyAllocations(secondAllocation);
+        mInfos = allocationManager.getAllocationInfo(defaultOperator, strategyMock, secondAllocation[0].operatorSets);
+        console.log("deallocation effect timestamp: ", deallocationEffectTimestamp);
+        console.log("allocation effect timestamp: ", allocationEffectTimestamp);
+        assertEq(allocationEffectTimestamp, mInfos[0].effectTimestamp, "effect timestamp not correct");
+        assertLt(allocationEffectTimestamp, deallocationEffectTimestamp, "invalid test setup");
+
+        // Warp to allocation effect timestamp & clear the queue
+        cheats.warp(allocationEffectTimestamp);
+        allocationManager.clearDeallocationQueue(defaultOperator, _strategyMockArray(), _maxNumToClear());
+
+        // Validate `getAllocatableMagnitude`. Allocatable magnitude should be the difference between the total magnitude and the encumbered magnitude
+        uint64 allocatableMagnitude = allocationManager.getAllocatableMagnitude(defaultOperator, strategyMock);
+        assertEq(WAD - 33e16 - 5e17, allocatableMagnitude, "allocatableMagnitude not correct");
+
+        // Validate that we can allocate again for opset2. This should not revert
+        IAllocationManagerTypes.MagnitudeAllocation[] memory thirdAllocation =
+            _generateMagnitudeAllocationCalldataForOpSet(defaultAVS, 2, 10e16, 1e18);
+        cheats.prank(defaultOperator);
+        allocationManager.modifyAllocations(thirdAllocation);
+    }
+
+    /**
+     * Allocates to opset1, allocates to opset2, deallocates from opset1. Asserts that the allocation, which has a higher
+     * effect timestamp is not blocking the deallocation.
+     * The queue looks like (note that the first allocation has been completed):
+     * 1. (allocation, opSet2, mag: 5e17, effectTimestamp: 50th day)
+     * 2. (deallocation, opSet1, mag: 0, effectTimestamp: 42.5 day)
+     */
+    function test_regression_allocationBlocksDeallocation() public {
```

Comment:
nit: testname sounds like we expect the allocation to block the deallocation.
Maybe `test_regression_deallocationNotBlocked`?

Reply:
Updated